estimator/transport_co2/model.py

```diff
@@ -42,16 +42,35 @@ def estimate_co2_g(
 
 class Mode(Enum):
     """
-    Data structure containing grams of CO2/vehicle KM for several modes of transport
-    along with average occupancy for each mode.
+    Tuples for several modes of transport, each containing:
+    - average grams of CO2 per vehicle kilometer
+    - average vehicle occupancy.
 
-    See the library README for sources and a description of how we arrive at these CO2 values.
+    The modes of transport come from the OpenTripPlanner project.
+
+    See the library README for data sources and a description of how we arrive at these CO2 values.
     """
 
     def __init__(self, avg_co2_per_vehicle_km: float, avg_occupancy: float):
         self.avg_co2_per_vehicle_km = avg_co2_per_vehicle_km
         self.avg_occupancy = avg_occupancy
 
+    def __calculate_mean_of_transport_modes(modes: list) -> tuple:
```

Comment:
Nice name indeed. 😎 

The two leading underscores [kinda behave weird](https://docs.python.org/3/tutorial/classes.html#private-variables). Is that on purpose?

Regardless of that, the method doesn't really need to be part of the class (`self` is neither declared nor used), so I'd think it's best to move it outside the class.

Reply:
Also, while `list` is fine as type annotation (after all there's a very limited scope to this method), the body works just the same with any `Iterable`, and I think it also makes sense semantically.

Reply:
> lists, tuples, and strings - any such sequence can be iterated over in a for-loop. We will also encounter important non-sequential collections, like dictionaries and sets

I am just not sure how we would handle cases like a list or dictionary being passed into the function.

Reply:
Marking this as resolved, since I have moved the function out of the Mode class.